web/src/index.css

```diff
@@ -28,3 +28,30 @@
       @apply text-lg font-semibold;
     }
 }
+
+.google_translate {
+  @apply fixed pt-16 right-2 z-30;
+  .goog-logo-link{
+    display:none !important;
+   }
+  .goog-te-combo {
+  color:rgb(100 116 139);
+  border-radius:0.375rem;
+  width:6rem;
+  @media (min-width: 768px) {
+    width: 100%;
+  }
+  }
+ .goog-te-gadget{
+  color:transparent!important;
+  }
+ .VIpgJd-ZVi9od-l4eHX-hSRGPd{
+   z-index: 9724790009779!important;
+   top:0;
+   left:unset;
+   right:-5px;
+   display:none!important;
+   border-radius:50%;
+   border:2px solid gold;
+  }
+}
```

Comment:
I would style it so that it appears in the menu bar and in the dashboard menu, because right now it looks kinda funky

![afbeelding](https://github.com/drikusroor/ohayo-goededagu/assets/8208970/ea91e0e7-3bd0-4bf2-9d5d-aa2b4e2c6199)

![afbeelding](https://github.com/drikusroor/ohayo-goededagu/assets/8208970/b2362cb6-f9ec-4831-a456-f9580a0420e3)

Also, I see the frontpage is crashing again (just like in my PR). Seems to be related to a missing profile..

![afbeelding](https://github.com/drikusroor/ohayo-goededagu/assets/8208970/47921482-726c-4153-b36f-1cbe2f905a83)

Reply:
It looks like google charges for using the translate button: https://cloud.google.com/translate/pricing
So maybe we better not use it, I've tried searching for something else but have not yet been able to find anything.

We could maybe still implement the hacky englishTitle & englishBody as a solution?

Reply:
How would they charge us though? I don't see any information in the PR that leads to our google account or credit cards ;-) I think the link that you are referring to is about using the Google Translate API, which we aren't using afaik. This is just a button that ends up as free advertising for google.

Reply:
That is true 🤔
how they gonna know